Alpha-61 and 7.7-rc-1: crash on start on a fresh install

```
06-20 05:44:59.119 2718-2718/org.wordpress.android.beta E/AndroidRuntime: FATAL EXCEPTION: main
  Process: org.wordpress.android.beta, PID: 2718
  java.lang.RuntimeException: Unable to create application org.wordpress.android.WordPressDebug: android.database.sqlite.SQLiteException: duplicate column name: sharingLabel (code 1): , while compiling: alter tablesite_settings add sharingLabel TEXT;
    at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4710)
    at android.app.ActivityThread.-wrap1(ActivityThread.java)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1405)
    at android.os.Handler.dispatchMessage(Handler.java:102)
    at android.os.Looper.loop(Looper.java:148)
    at android.app.ActivityThread.main(ActivityThread.java:5417)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
 Caused by: android.database.sqlite.SQLiteException: duplicate column name: sharingLabel (code 1): , while compiling: alter table site_settings add sharingLabel TEXT;
    at android.database.sqlite.SQLiteConnection.nativePrepareStatement(Native Method)
    at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:887)
    at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:498)
    at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:588)
    at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:58)
    at android.database.sqlite.SQLiteStatement.<init>(SQLiteStatement.java:31)
    at android.database.sqlite.SQLiteDatabase.executeSql(SQLiteDatabase.java:1674)
    at android.database.sqlite.SQLiteDatabase.execSQL(SQLiteDatabase.java:1605)
    at org.wordpress.android.datasets.SiteSettingsTable.addSharingColumnsToSiteSettingsTable(SiteSettingsTable.java:51)
    at org.wordpress.android.WordPressDB.<init>(WordPressDB.java:211)
    at org.wordpress.android.WordPress.initWpDb(WordPress.java:446)
    at org.wordpress.android.WordPress.onCreate(WordPress.java:257)
    at org.wordpress.android.WordPressDebug.onCreate(WordPressDebug.java:13)
    at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1013)
    at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4707)
    at android.app.ActivityThread.-wrap1(ActivityThread.java) 
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1405) 
    at android.os.Handler.dispatchMessage(Handler.java:102) 
    at android.os.Looper.loop(Looper.java:148) 
    at android.app.ActivityThread.main(ActivityThread.java:5417) 
    at java.lang.reflect.Method.invoke(Native Method) 
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
```

--- a/WordPress/src/main/java/org/wordpress/android/models/SiteSettingsModel.java
+++ b/WordPress/src/main/java/org/wordpress/android/models/SiteSettingsModel.java
@@ -116,13 +116,7 @@ public class SiteSettingsModel {
                     USER_ACCOUNT_REQUIRED_COLUMN_NAME + " BOOLEAN, " +
                     WHITELIST_COLUMN_NAME + " BOOLEAN, " +
                     MODERATION_KEYS_COLUMN_NAME + " TEXT, " +
-                    BLACKLIST_KEYS_COLUMN_NAME + " TEXT," +
-                    SHARING_LABEL_COLUMN_NAME + " TEXT," +
-                    SHARING_BUTTON_STYLE_COLUMN_NAME + " TEXT," +
-                    ALLOW_REBLOG_BUTTON_COLUMN_NAME + " BOOLEAN," +
-                    ALLOW_LIKE_BUTTON_COLUMN_NAME + " BOOLEAN," +
-                    ALLOW_COMMENT_LIKES_COLUMN_NAME + " BOOLEAN," +
-                    TWITTER_USERNAME_COLUMN_NAME + " TEXT" +
+                    BLACKLIST_KEYS_COLUMN_NAME + " TEXT" +
                     ");";
 
     public boolean isInLocalTable;

